client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -291,7 +291,7 @@ class Security2faBackupCodesList extends React.Component {
 						onClick={ this.onNextStep }
 						disabled={ this.isSubmitDisabled() }
 					>
-						{ this.props.translate( 'All Finished!', {
+						{ this.props.translate( 'All finished!', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.